src/test/java/org/cactoos/scalar/FallbackFromTest.java

```diff
@@ -26,52 +26,51 @@
 import java.io.IOException;
 import java.util.IllegalFormatException;
 import org.cactoos.iterable.IterableOf;
-import org.hamcrest.MatcherAssert;
 import org.hamcrest.core.IsEqual;
 import org.junit.Test;
+import org.llorllale.cactoos.matchers.Assertion;
 
 /**
  * Test case for {@link FallbackFrom}.
  *
  * @since 0.31
  * @checkstyle JavadocMethodCheck (500 lines)
  */
-@SuppressWarnings("unchecked")
 public final class FallbackFromTest {
 
     @Test
     public void supportsException() {
-        MatcherAssert.assertThat(
-            "IOException class is not supported, but should be",
+        new Assertion<>(
+            "Must support IOException class",
             new FallbackFrom<>(
                 new IterableOf<>(IOException.class),
                 exp -> "IOException fallback"
             ).support(IOException.class),
             new IsEqual<>(0)
-        );
+        ).affirm();
     }
 
     @Test
     public void supportsInheritedException() {
-        MatcherAssert.assertThat(
-            "RuntimeException class is not supported, but should be",
+        new Assertion<>(
+            "Must support RuntimeException class",
             new FallbackFrom<>(
                 new IterableOf<>(RuntimeException.class),
```

Comment:
@victornoel error log: `unchecked generic array creation for varargs parameter of type java.lang.Class<? extends java.lang.Throwable>[]`